tests/VolleyManagement.UnitTests/Services/GameService/GameServiceTests.cs

```diff
@@ -1414,7 +1415,7 @@ public void GetTournamentResults_GameResultsRequsted_GameResultsReturned()
             var actual = sut.GetTournamentResults(TOURNAMENT_ID);
 
             // Assert
-            CollectionAssert.AreEqual(expected, actual, new GameResultDtoComparer());
+            CollectionAssert.AreEqual(expected, actual as ICollection, new GameResultDtoComparer());
```

Comment:
cast

Reply:
there is no exit from this type of problems, cast in one part of code or in another, mb there is some unknown to me method?

Reply:
Casts in assertions are bad for maintainability

Reply:
okey